src/reverseRegistrar.ts

```diff
@@ -0,0 +1,18 @@
+import { ReverseClaimed } from "./types/ReverseRegistrar/ReverseRegistrar";
+import { Account, Domain, ReverseClaimed as ReverseClaimedEvent } from "./types/schema";
+import { createEventID } from './utils'
+
+export function handleReverseClaimed(event: ReverseClaimed): void {
+    const account = new Account(event.params.addr.toHex())
+    const domain = new Domain(event.params.node.toHex())
+    domain.save()
+    account.reverseDomain = domain.id
```

Comment:
If we're doing this, we should a) Check if the name actually resolves (eg, if the forward resolution matches), and b) Invalidate it if the forward resolution changes.